src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        String randomColor = ColorSupplier.getRandomColor();
+        Random random = new Random();

```

Comment:
Make random the class level field
```suggestion
        Random random = new Random();

```